runtime/profile/_/mod/core/locale/jp/config.hcl

```diff
@@ -492,6 +492,9 @@ DOC
             keybindings {
                 name = "キーの割り当て"
             }
+            mods {
+                name = "モッド設定"
```

Comment:
The translation of "mod" and "mods" in Japanese is "MOD" (always uppercase and singular).